src/static/riot/analytics/analytics.tag

```diff
@@ -0,0 +1,540 @@
+<analytics>
+    <h1>Analytics</h1>
+
+
+    <h3>Date Range</h3>
+    <div class="ui blue buttons" ref="shortcut_buttons">
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-week">This Week</button>
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-month">This Month</button>
+        <button class="ui button" onclick="{ time_range_shortcut }" id="this-year">This Year</button>
+        <button class="ui button" onclick="{ show_date_selectors }" ref="custom_date">Custom</button>
+    </div>
+
+    <div class='hidden date-selection' ref="date_selection_container">
+        <div class="start-date-input date-input">
+            <h3>Start Date</h3>
+            <div class="ui calendar" ref="start_calendar">
+                <div class="ui input left icon">
+                    <i class="calendar icon"></i>
+                    <input type="text" placeholder="{ start_date }">
+                </div>
+            </div>
+        </div>
+        <div class="date-input">
+            <h3>End Date</h3>
+            <div class="ui calendar" ref="end_calendar">
+                <div class="ui input left icon">
+                    <i class="calendar icon"></i>
+                    <input type="text" placeholder="{ end_date}">
+                </div>
+            </div>
+        </div>
+    </div>
+
+    </div>
+
+    <h3>Chart Resolution</h3>
+    <div class="ui blue buttons">
+        <button class="ui button" onclick="{ update_chart_resolution }" id="day-units">Daily</button>
+        <button class="ui button" onclick="{ update_chart_resolution }" id="week-units">Weekly</button>
+        <button class="ui button" onclick="{ update_chart_resolution }" id="month-units">Monthly</button>
+    </div>
+
+
+    <div class="ui top attached tabular menu">
+        <a class="active item" data-tab="competitions">Competitions</a>
+        <a class="item" data-tab="submissions">Submissions</a>
+        <a class="item" data-tab="users">Users</a>
+    </div>
+
+        <div class="ui bottom attached active tab segment" data-tab="competitions">
+            <div class="ui statistics">
```

Comment:
Maybe make these `ui small statistics` so the numbers aren't _so_ big

Reply:
Done.